external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaOffsetReader.scala

```diff
@@ -447,7 +422,7 @@ private[kafka010] class KafkaOffsetReader(
     }
   }
 
-  private def getSortedExecutorList(): Array[String] = {
+  private def getSortedExecutorList: Array[String] = {
```

Comment:
This looks to be unnecessary change, though it doesn't look to have side effect so semantically OK.

Reply:
Yeah, this is one of the `minor bugfixes here and there`.

Reply:
Leaving this open for others.